main/arbitration_identifiers.h

```diff
@@ -0,0 +1,8 @@
+#include <stdint.h>
+
+static uint32_t send_identifier = 0x7E0; // ECU
```

Comment:
i could see this needing a semaphore or us needing something more than a 1:1 relation

Reply:
I think the best way to accomplish this would be to encapsulate the state of the isotp library in an enumeration and have multiple instances of the library. otherwise we'll get all of the timing variables and state crossed up between modules. unfortunately this is a fair amount of work and I don't have a need for multiple module support, so I'm not inclined to do it off the bat.

Reply:
I think we'd need to encapsulate the internal state of the isotp stack and create multiple instances of it to support switching control modules on the fly / multiple sessions at once - otherwise, stuff like the timeouts and flow control state will get crossed between control modules.